v1/util/formatter_util.cc

```diff
@@ -0,0 +1,37 @@
+#include "formatter_util.h"
+
+namespace cloudevents {
+namespace formatter_util {
+
+using ::cloudevents::format::Format;
+using ::cloudevents::format::Formatter;
+using ::cloudevents::format::JsonFormatter;
+
+absl::StatusOr<Format> FormatterUtil::DestringifyFormat(
+        const std::string& format_str) {
+    if (format_str == "json") {
+        return Format::kJson;
+    }
+    return absl::InvalidArgumentError("The given format is not currently recognized by the SDK.");
+}
+
+std::string FormatterUtil::StringifyFormat(const Format& format) {
+    switch (format) {
+        case Format::kJson:
+            return std::string("json");
+    }
+}
+
+absl::StatusOr<
+        std::unique_ptr<Formatter>> 
+        FormatterUtil::GetFormatter(const Format& format) {
+    switch (format) {
+        case Format::kJson:
+            auto m = new JsonFormatter;
+            return std::unique_ptr<Formatter>(m);
```

Comment:
combine this line with the one above.

Reply:
Same issue as above with StatusOr and polymorphism between Formatter and JsonFormatter